src/software/api/codebase/api.js

```diff
@@ -33,16 +33,15 @@
  */
 var GeotabApi = function (getCredentialsCallback, newOptions, customCredentialStore) {
     "use strict";
-    var
-        JSONP_REQUESTS_PROPERTY_STR = "geotabJSONP",
+    var JSONP_REQUESTS_PROPERTY_STR = "geotabJSONP",
         credentials,
         server,
         pendingCalls = [],
         options = {
             // Use localStorage to store credentials automatically
             rememberMe: true,
             // Show debugging information (in Chrome dev tools, Firebug, etc.)
-            debug: false,
+            debug: true,
```

Comment:
Do we need this on or was just for local testing?

Reply:
Well, I figured that since this tool is mostly used for development that it made sense to enable logging for it. I can turn disable it if you want.